docs/configuration-reference/components/cert-manager.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Cert-Manager configuration reference for Lokomotive
+title: cert-manager configuration reference for Lokomotive
```

Comment:
Nit: this could go into separate commit.

Reply:
Done!